app/views/posts/_comment.html.erb

```diff
@@ -1,7 +1,12 @@
-<p class="username"><%= com.user.email %> <%= time_ago_in_words(com.created_at) %> ago</p>
+<p class="username"><%= com.votes.sum("vote_type") %> karma by <%= com.user.email %> <%= time_ago_in_words(com.created_at) %> ago</p>
```

Comment:
This is another thing that could be moved to an instance method on comment.

Reply:
Returning the whole string?

Reply:
No, just the vote count. Maybe something like `comment.vote_count` or comment.total_votes` or whatever name makes the most sense to you.